docs/guide/index.md

```diff
@@ -65,10 +65,10 @@ You can also directly specify the project name and the template you want to use
 
 ```bash
 # npm 6.x
-npm init vite@latest my-vue-app --template vue
+npm create vite@latest my-vue-app --template vue
```

Comment:
I'm not sure if npm 6 also have this alias, could you check?

Reply:
I tried with NPM 6.14.15 and found it possible